src/options/pages/onboarding/SetupPage.test.tsx

```diff
@@ -106,4 +111,77 @@ describe("SetupPage", () => {
 
     expect(screen.getByText("Connect your AARI account")).not.toBeNull();
   });
+
+  test("Start URL", async () => {
+    // User will be unauthenticated
+    (useGetMeQuery as jest.Mock).mockImplementation(() => ({
+      isLoading: false,
+      data: {},
+    }));
+
+    (global as any).window = Object.create(window);
+
+    const url = new URL(
+      "chrome-extension://abc/options.html?hostname=https://mycontrolroom.com#/start"
```

Comment:
- [ ] @mnholtz can you confirm this is the URL format? In particular, that the hostname is a search param on the main URL, not part of the hash?

Reply:
The hostname is actually a search param on the hash https://github.com/pixiebrix/pixiebrix-extension/blob/b5c30fd68bd3180fccf3c3360f79033e21b86821/src/background/installer.ts#L53-L53
```suggestion
      "chrome-extension://abc/options.html#/start?hostname=https://mycontrolroom.com"
```

Reply:
@mnholtz isn't this check wrong then? https://github.com/pixiebrix/pixiebrix-extension/blob/dda30f7c83ee1e04c90e8306a85613568fb6e977/src/options/pages/onboarding/partner/PartnerSetupCard.tsx#L104-L104